### ISSUE TITLE
Unable to specify what rules to validate against

### ISSUE THREAD

**Issue by aarondaub** [realm/SwiftLint#1]
We should be able to specify what to use - or perhaps specify what rules NOT to use. We should also probably be able to get a list of current rules.

**Comment by aarondaub** [comment]
I'll open a PR for this when I'm back at my laptop this evening.

**Comment by jpsim** [comment]
That would be great! The design I'm considering is to have a `.swiftlint.yml` file or similar (I've opened up #2 to track that).

I'd also like a syntax like
```swift
// swiftlint:disable_rule:line_length
... some long line
// swiftlint:enable_rule:line_length
```

I've opened up #4 to track *that*. PRs are definitely welcome, or even discussion around the design.

**Comment by aarondaub** [comment]
I'm having an issue constraining objects to Rule because it has a type alias. Trying to think of a good solution.

**Comment by eneko** [comment]
I support the idea of a `.swiftlint.yml` configuration file to disable or configure some rules, like line_length. Any work completed on that yet?

In addition, I'd like to suggest for the same configuration file to include options to specify which project folders to include or exclude. This will allow for easy excluding of Pods or third party libraries that developers shouldn't be concerned about (unless working on those projects).
